### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by Oman395** [SebLague/Digital-Logic-Sim#152] [General]
Hey, almost done with my AUR port-- is there any logo I can use for the icon, or should I make my own?

**Comment by Oman395** [comment]
Currently uploaded to AUR, I just made a shitty icon that works for now

**Comment by SebLague** [comment]
Hi, I've been using [this](https://github.com/SebLague/Digital-Logic-Sim/blob/main/Assets/Modules/Chip%20Creation/Graphics/Icons/App%20Icon.psd) for the icon, but it's just temporary until I get around to making something better!

**Reply by Oman395** [reply]
Thanks, package is updated!
